source/community/reactnative/src/components/elements/rich-text-block.js

```diff
@@ -42,9 +42,7 @@ export class RichTextBlock extends React.Component {
         if (this.payload.paragraphs) {
             this.payload.paragraphs.forEach((paragraph, index) => {
```

Comment:
forEach can be replaced map.

Reply:
It's part of old code. we can change it later